packages/peregrine/lib/talons/WishlistPage/createWishlist.gql.js

```diff
@@ -0,0 +1,38 @@
+import { gql } from '@apollo/client';
+
+export const CREATE_WISHLIST = gql`
+    mutation createWishlist(
+        $name: String!
+        $visibility: WishlistVisibilityEnum!
+    ) {
+        createWishlist(input: { name: $name, visibility: $visibility }) {
```

Comment:
Minor preference that makes changes easier later, model your form after the mutation input, and just pass it as-is.

```suggestion
    mutation CreateWishlist(
        $input: CreateWishlistInput!
    ) {
        createWishlist(input: $input) {
```

Reply:
Changed.
Thank you